man/mono.1

```diff
@@ -588,6 +588,11 @@ Controls whether the runtime should attempt to inline (the default),
 or not inline methods invocations
 .ne
 .RE
+\fB--response=FILE\fR
```

Comment:
@migueldeicaza I'd prefer if we made this `@file.rsp` to match the convention set by roslyn/csc and other tools: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-options/response-file-compiler-option